articles/native-platforms/ios-objc/06-rules.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Rules
+description: This tutorial will show you how to create a basic rule that you can use in your app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objc',
+  pkgFilePath: 'iOS/basic-sample-objc/ObjcSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
+
+### 1. Create a Rule
+
+To create a rule, just go to the [new rule page](https://manage.auth0.com/#/rules/new). You can create it from scratch or use an existing template. These templates are written by Auth0 team to assist you complete common tasks.
+
+Let's use the template called "*Add country to the user profile*", under the *Enrich Profile* section:
+
+![Add country template](/media/articles/rules/rule-choose-add-country-template.png)
```

Comment:
The image size is 1212 × 853. Please resize to max 750 width. 

Also, make sure that you run all images through [TinyPNG](https://tinypng.com/).